alf/algorithms/data_transformer.py

```diff
@@ -267,7 +267,7 @@ def _make_state(self, spec):
 
     def transform_timestep(self, time_step, state):
         if self._stack_size == 1:
-            return time_step
+            return time_step, ()
```

Comment:
Why did you change this return?

Reply:
Good point.  Probably doesn't matter as there is no framestack when n==1.  Returning state just in case.